internal/util/csiconfig.go

```diff
@@ -39,11 +39,8 @@ const (
 // ClusterInfo strongly typed JSON spec for the below JSON structure.
 type ClusterInfo struct {
 	// ClusterID is used for unique identification
-	ClusterID string `json:"clusterID"`
-	// RadosNamespace is a rados namespace in the pool
-	RadosNamespace string `json:"radosNamespace"` // For backward compatibility. TODO: Remove this in 3.7.0
-	// Monitors is monitor list for corresponding cluster ID
-	Monitors []string `json:"monitors"`
+	ClusterID string   `json:"clusterID"`
+	Monitors  []string `json:"monitors"`
```

Comment:
Is there a good reason to remove the comment for `Monitors`?

Reply:
its a overside, fixed it now